pkg/utils/helper.go

```diff
@@ -11,6 +12,64 @@ import (
 	"strings"
 )
 
+// PublicAzureStorageEndpointSuffix defines default Storage Endpoint Suffix
+const PublicAzureStorageEndpointSuffix = "core.windows.net"

```

Comment:
That would work fine but I am wondering if the user should specify the whole hostname (rather than just the name today) so that we don't have to determine it by ourselves. What do you think?

Reply:
@arnaud-tincelin I'm a bit confused by what you mean - how would the user pass it in? Would it be through the yaml file or some other method?

Reply:
Yes indeed as of today the deployment requires the following secret:
```yaml
---
apiVersion: v1
kind: Secret
metadata:
  name: azureblob-secret
  namespace: aks-periscope
type: Opaque
data:
  AZURE_BLOB_ACCOUNT_NAME: YXRpcGVyaXNjb3Bl
```
we could change it to `AZURE_BLOB_ACCOUNT_HOSTNAME` or `AZURE_BLOB_ACCOUNT_BASE_URL`

PS: this is a simple question, I'm not requesting changes :)